### DISCUSSION TITLE
Hide Cancel Button

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#4745] [Help]
Is it possible to hide the cancel button from the upload and edition forms?

**Comment by AnderssonGt** [comment]
this will hide the action buttons
``->modalActions([])``


Example:
`Tables\Actions\ViewAction::make('Detalles de pago')
                ->modalContent(fn ($record) => view('livewire.payment-detail', ['record' => $record]))
                ->modalHeading('Detalles de pago')
                ->icon('grommet-view')
                ->color('success')
                ->label('')
                ->tooltip('Detalles de pago')                
                ->modalActions([]),`

**Comment by putricha** [comment]
You can hide the cancel button from the upload and edition forms by using the `->modalCancelAction(false)` method, which disables the cancel action on the modal, ensuring the button is not displayed.
